generic-database-plugin/src/main/java/co/cask/jdbc/DatabaseSource.java

```diff
@@ -43,20 +43,10 @@ public DatabaseSource(DatabaseSourceConfig databaseSourceConfig) {
     this.databaseSourceConfig = databaseSourceConfig;
   }
 
-  @Override
-  public String getJdbcDriverName() {
-    return Objects.nonNull(databaseSourceConfig) ? databaseSourceConfig.jdbcDriverName : null;
-  }
-
   /**
    * Generic database source configuration.
    */
   public static class DatabaseSourceConfig extends DBSourceConfig {
-    @Name(DatabaseConstants.JDBC_DRIVER_NAME)
-    @Description("Name of the JDBC driver to use. This is the value of the 'jdbcDriverName' key defined in the JSON " +
-            "file for the JDBC plugin.")
-    public String jdbcDriverName;
```

Comment:
why is the jdbc driver getting removed from all the DB configs?

Reply:
It's moved back to ConnectionConfig because you and I decided to keep this field for DB-specific plugins (with default value pre-populated). So now it is used by all configurations.